### PR TITLE
Add Russian Ember community writeup

### PR DESCRIPTION
**Pull request by amyrlam**
FYI @chilicoder @lifeart, if you have any feedback!

### CODE REVIEW

File: source/2020-01-17-the-ember-times-issue-131.md
Comment (ijlee2) [inline source/2020-01-17-the-ember-times-issue-131.md]: Did we want to link the title to a website?

**Review by ijlee2 — APPROVED** [review]
Looks good to me, thanks for covering this! I think it'd be good to wait for feedback from Iaroslav or Alex before merging.

File: source/2020-01-17-the-ember-times-issue-131.md
Comment (amyrlam) [reply source/2020-01-17-the-ember-times-issue-131.md]: whoops, added link to issue # 130. yeah not sure about the title. we'll see if they chime in

Comment (yarigpopov) [reply source/2020-01-17-the-ember-times-issue-131.md]: I think the title is a proper one.
Thanks @amyrlam for cleaning the message. That would be exciting to translate)

Just a minor thing. Comma after Привет to conform Russian grammar.
https://github.com/ember-learn/ember-blog/pull/495/files

Also, if you find it neat (and understandable) enough, here how to write the whole greeting in Russian:

Привет, Эмберисты!

Comment (amyrlam) [reply source/2020-01-17-the-ember-times-issue-131.md]: thanks! i added "Hello, Emberistas!" in italics so it will make sense 👍

**Comment by amyrlam** [comment]
thanks @ijlee2 @chilicoder!

**Pull request merged**
